services/integration/src/model/projectModel.js

```diff
@@ -0,0 +1,40 @@
+const mongoose = require('mongoose');
```

Comment:
change this file name for project.model.js

Reply:
good . It's already changed.